cmd/subctl/unexport.go

```diff
@@ -61,7 +61,7 @@ var (
 )
 
 func init() {
-	unexportRestConfigProducer.SetupFlags(unexportCmd.Flags())
+	unexportRestConfigProducer.SetupFlags(unexportServiceCmd.Flags())
```

Comment:
The correct action would be to make `unexport` command flags `persistent` so that they apply to all sub commands as well.

```suggestion
	unexportRestConfigProducer.SetupFlags(unexportCmd.PersistentFlags())
```

Reply:
Thanks for your suggestion, Jaanki.